Add native and erc20 fet section (#256)
fetchai/docs: docs/native_and_erc20/native_and_erc20_fet.md

The **Fetch.ai token (FET)** is a utility token and the key medium of exchange on the Fetch.ai network. FET can be used to pay for services in the Fetch ecosystem and network transaction fees. Users can also choose to stake FET to participate in securing the network via its Proof-of-Stake consensus mechanism and earn rewards in return for contributing to validator nodes. FET tokens can be purchased in different [centralized or decentralized exchanges](https://fetch-ai.network/get-fet/). 

> The Fetch.ai team initially developed the FET utility token on an ERC-20 contract on the Ethereum network while finishing the work on the Fetch.ai native main network (main-net). This had the benefit of allowing helped for developing innovative solutions and helped developers across the world to get their hands on the Fetch.ai ecosystem. However, Ethereum is not capable of scaling in the way necessary to allow Fetch.ai to be truly scalable. Think of the high gas fees you face when executing a transaction on the Ethereum Mainnet. This is the reason which led to the creation of the native FET token.

Reviewer: Link `ERC-20 contract on the Ethereum network` to the contract.

The **Fetch.ai token (FET)** is a utility token and the key medium of exchange on the Fetch.ai network. FET can be used to pay for services in the Fetch ecosystem and network transaction fees. Users can also choose to stake FET to participate in securing the network via its Proof-of-Stake consensus mechanism and earn rewards in return for contributing to validator nodes. FET tokens can be purchased in different [centralized or decentralized exchanges](https://fetch-ai.network/get-fet/). 

> The Fetch.ai team initially developed the FET utility token on an ERC-20 contract on the Ethereum network while finishing the work on the Fetch.ai native main network (main-net). This helped developers across the world kickstart the process of developing innovative solutions in the Fetch.ai ecosystem. However, the Ethereum chain does not offer the scalability needed by the kinds of applications the fetch.ai ecosystem aims for. For example, think of the high gas fees you face when executing a transaction on the Ethereum Main-net. This lead to the creation of the native FET token.

Reviewer: Link `ERC-20 contract on the Ethereum network` to the contract.

> The Fetch.ai team initially developed the FET utility token on an ERC-20 contract on the Ethereum network while finishing the work on the Fetch.ai native main network (main-net). This helped developers across the world kickstart the process of developing innovative solutions in the Fetch.ai ecosystem. However, the Ethereum chain does not offer the scalability needed by the kinds of applications the fetch.ai ecosystem aims for. For example, think of the high gas fees you face when executing a transaction on the Ethereum Main-net. This lead to the creation of the native FET token.

Reviewer: ```suggestion
The Fetch.ai team initially developed the FET utility token on an ERC-20 contract on the Ethereum network while finishing the work on the Fetch.ai native main network (main-net). This helped developers across the world get their hands on the FET tokens earlier and kickstart the process of developing innovative solutions within the Fetch.ai ecosystem. FET, as an ERC20 token on the Ethereum contract has never been Fetch.ai's end game because the Ethereum chain does not offer the degree of scalability needed by the kinds of applications the fetch.ai ecosystem aims for. For example, think of the high gas fees you face when executing a transaction on the Ethereum main-net. 
```

> The Fetch.ai team initially developed the FET utility token on an ERC-20 contract on the Ethereum network while finishing the work on the Fetch.ai native main network (main-net). This helped developers across the world get their hands on the FET tokens earlier and kickstart the process of developing innovative solutions within the Fetch.ai ecosystem. FET, as an ERC20 token on the Ethereum contract has never been Fetch.ai's end game because the Ethereum chain does not offer the degree of scalability needed by the kinds of applications the fetch.ai ecosystem aims for. For example, think of the high gas fees you face when executing a transaction on the Ethereum main-net. 

Reviewer: link `ERC-20 contract on the Ethereum network` to the erc20 contract.

With the launch of the **Fetch.ai Main-net**, users operating on the Fetch.ai network no longer need to hold any other token (e.g. ETH or BTC) associated with any other network. It is only the native FET tokens that fuel the Fetch.ai ecosystem and its applications.

> 

Reviewer: Add a paragraph explaining that:

- Today both erc20 and native FETs are in circulation
- For the foreseeable future, both will live side by side.
- Certain exchanges buy/sell on type or the other.

> 

Reviewer: Feel free to bring back the block explorer links (either bring the link into the text or as a separate section like it was but where it clearly shows the value of those links)

> _ERC-20 FET token holders can easily convert their ERC-20 FET tokens into native ones and vice versa via the **Fetch.ai Token Bridge**, available at: <https://token-bridge.fetch.ai/>._

Reviewer: ```suggestion
Learn [how to convert ERC-20 and native FETs to each other](../how_to_convert_fet)
```

> Learn [how to convert ERC-20 and native FETs to each other](../how_to_convert_fet)

Reviewer: ```suggestion
Learn [how to convert ERC-20 and native FETs to each other](../how_to_convert_fet).
```

> The **Fetch.ai token (FET)** is an utility token and the key medium of exchange on the Fetch.ai network. FET can be used to pay for services in the Fetch ecosystem and network transaction fees. Users can also choose to stake FET to participate in securing the network via its Proof-of-Stake (PoS) consensus mechanism and earn rewards in return for contributing to validator nodes. 

Reviewer: Remember to link the ...`Proof-of-Stake (PoS) consensus mechanism`... with the consensus/proof-of-stake page if we create such a page later on.

> The Fetch.ai team initially developed the FET utility token on an [ERC-20 contract](https://etherscan.io/token/0xaea46A60368A7bD060eec7DF8CBa43b7EF41Ad85) on the Ethereum network while finishing the work on the Fetch.ai native main network (main-net). This helped developers across the world get their hands on the FET tokens earlier and kickstart the process of developing innovative solutions within the Fetch.ai ecosystem. FET, as an ERC-20 token on the Ethereum contract has never been Fetch.ai's end game because the Ethereum chain does not offer the degree of scalability needed by the kinds of applications the fetch.ai ecosystem aims for. For example, think of the high gas fees you face when executing a transaction on the Ethereum main-net.  

Reviewer: Remove the last line: `For example, think`...

With the launch of the **Fetch.ai Main-net**, the native FET became available. This meant that users operating on the Fetch.ai network no longer needed to hold any token (e.g. ETH or BTC) associated with any other network. It is only the native FET tokens that fuel the Fetch.ai ecosystem and its applications.

> ERC-20 FET tokens are still in circulation and currently co-exist with native FET tokens and can be transferred from one to the other easily on the Fetch.ai network. All of FET ERC-20 tokens will ultimately become native FET tokens, but in the foreseeable future both will live side by side. FET tokens can be purchased from different [centralized or decentralized exchanges](https://fetch-ai.network/get-fet/). However, certain exchanges may buy or sell one type of FET token or the other. 

Reviewer: ```suggestion
ERC-20 FET tokens are still in circulation and currently co-exist with native FET tokens and can be transferred from one to the other easily on the Fetch.ai network. All of the ERC-20 FET tokens will ultimately become native FET tokens, but in the foreseeable future both will live side by side. FET tokens can be purchased from different [centralized or decentralized exchanges](https://fetch-ai.network/get-fet/). However, certain exchanges may buy or sell one type of FET token or the other. 
```

> ERC-20 FET tokens are still in circulation and currently co-exist with native FET tokens and can be transferred from one to the other easily on the Fetch.ai network. All of FET ERC-20 tokens will ultimately become native FET tokens, but in the foreseeable future both will live side by side. FET tokens can be purchased from different [centralized or decentralized exchanges](https://fetch-ai.network/get-fet/). However, certain exchanges may buy or sell one type of FET token or the other. 

Reviewer: Link ...`transferred from one to the other`... to the next page. Remove the last line (it is dangling).
